src/main/java/fish/payara/extras/upgrade/BaseUpgradeCommand.java

```diff
@@ -225,11 +225,9 @@ protected boolean reinstallSSHNode(Node node) {
         LOGGER.log(Level.INFO, "Reinstalling SSH node {0}", new Object[]{node.getName()});
         ArrayList<String> command = new ArrayList<>();
         command.add(SystemPropertyConstants.getAdminScriptLocation(glassfishDir));
+        command.add("install-node-ssh");
```

Comment:
Why have you moved this? 
The format of commands should be:
`asadmin ${asadminOptions} ${command} ${commandOptions}`